docs/tutorial-plain.rst

```diff
@@ -1,12 +1,11 @@
-Introduction tutorial - Graphene and Django
+Basic Tutorial
 ===========================================
 
 Graphene has a number of additional features that are designed to make
-working with Django *really simple*.
-
-Our primary focus here is to give a good understanding of how to connect models from Django ORM to graphene object types.
-
-A good idea is to check the `graphene <http://docs.graphene-python.org/en/latest/>`__ documentation first.
+working with Django *really simple*. Our primary focus here is to give
+a good understanding of how to connect models from Django ORM to graphene
+object types. A good idea is to check the `graphene <http://docs.graphene-python.org/en/latest/>`__
+documentation first, but it is not essential to understand and use graphene-django in your project.
```

Comment:
I think we can drop this whole paragraph - it's not adding anything to a beginner who's starting out.